ncm-spma/src/main/pan/components/spma/yumng/schema.pan

```diff
@@ -0,0 +1,73 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+
+declaration template components/spma/yumng/schema;
+
+include 'components/spma/schema';
+
+type SOFTWARE_GROUP = {
+    "default" : boolean = true
+    "mandatory" : boolean = true
+    "optional" : boolean = false
+    "names" : string[]
```

Comment:
As per the comment on config.pan above, let's make ```"names" : string[] = list()``` to avoid having to set the default in config.pan